docs/dive-in/2024/01/2024-01-16:update-make-file.md

```diff
@@ -0,0 +1,14 @@
+# Make
+
+Make is a utility to automate somme commands.  
+Also it allows to have a kind of consistency between commands or to simplify commands.  
+It allows with a single command to run that will run multiple commands.
```

Comment:
Sentence unclear, can you reformulate?

Reply:
You can with a single command execute multiple command 

for exemple make start-then-stop:
 could execute docker compose up then docker compose down (even though this command would be the definition of useless :) )

Reply:
Pushed a new version with a better exemple (wasn't too difficult 😂)